app/models/package.scala

```diff
@@ -139,6 +139,7 @@ package object models {
                 }
               }
           }
+        case (_ , _) => JsError("path cannot be empty")
```

Comment:
This error should be "Invalid path and JSValue"

Reply:
changes made as per comment